features/machine/OCP24363.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Machine taints
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-24363
+  @admin
+  @destructive
+  Scenario Outline: [MAO] Reconciling machine taints with nodes
```

Comment:
You are using scenario outline for a single test case, I don't know if we support this format using polarion. It had better be tested using a runner-v3 job

Reply:
ok will test using v3